ringpop.go

```diff
@@ -646,21 +648,28 @@ func (rp *Ringpop) ringEvent(e interface{}) {
 }
 
 // GetReachableMembers returns a slice of members currently in this instance's
-// membership list that aren't faulty.
-func (rp *Ringpop) GetReachableMembers() ([]string, error) {
+// active membership list.
+func (rp *Ringpop) GetReachableMembers(predicates ...swim.MemberPredicate) ([]string, error) {
 	if !rp.Ready() {
 		return nil, ErrNotBootstrapped
 	}
-	return rp.node.GetReachableMembers(), nil
+
+	members := rp.node.GetReachableMembers(predicates...)
+
+	addresses := make([]string, 0, len(members))
+	for _, member := range members {
+		addresses = append(addresses, member.Address)
+	}
+	return addresses, nil
 }
 
 // CountReachableMembers returns the number of members currently in this
-// instance's membership list that aren't faulty.
```

Comment:
maybe change this to the more abstract `.. list that have a status that makes them reachable (e.g. not faulty)`